ui/page/settingsStripeCard/view.jsx

```diff
@@ -395,9 +416,29 @@ class SettingsStripeCard extends React.Component<Props, State> {
       setPreferredCurrency(value);
     }
 
+    function onChangeCardName(event) {
+      const { value } = event.target;
+
+      const numberOrSpecialCharacter = /[0-9`!@#$%^&*()_+\-=[\]{};':"\\|,.<>/?~]/;
```

Comment:
- [ ] Is this overlimiting?  Was expecting the following to be valid names
  - Gopal s/o Raju
  - Hubert Lord Sr.
  - James Wolfe-Blain
- [ ] Add character count limit like an actual card, then we don't need unnecessary css.

<img width="721" alt="image" src="https://user-images.githubusercontent.com/64950861/172976736-33c75fb9-4dd3-4ae6-8afe-c75e7c455a2c.png">

<img width="395" alt="image" src="https://user-images.githubusercontent.com/64950861/172977174-9114c5d1-030d-4768-90fe-383bdc79e192.png">